src/Elastic.Apm.SerilogEnricher/ElasticApmEnricherExtension.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using Serilog;
+using Serilog.Configuration;
+
+namespace Elastic.Apm.SerilogEnricher
+{
+
+	public static class ElasticApmEnricherExtension
+	{
+		/// <summary>
+		/// Enrich log events with a trace and transaction id properties containing the
+		/// current ids that the Elastic APM .NET Agent generated.
+		/// </summary>
+		/// <param name="enrichmentConfiguration">Logger enrichment configuration.</param>
+		/// <returns>Configuration object allowing method chaining.</returns>
+		/// <exception cref="ArgumentNullException">If <paramref name="enrichmentConfiguration"/> is null.</exception>
+		public static LoggerConfiguration WithElasticApmTraceId(
```

Comment:
Since it doesn't just add trace ID maybe a better name would be something like `WithElasticApmCorrelationInfo`?

Reply:
👍 `WithElasticApmCorrelationInfo` seems better. How about just `WithElasticApm()` ?

Reply:
I think `WithElasticApm` might be too broad. For example, some might infer from it that we can generate APM events from log entries (we have a few requests to support this use case).

Reply:
Changed it to `WithElasticApmCorrelationInfo`